### PR TITLE
add DJED and iUSD tokens; rename items according to CIP-36 changes

### PR DESCRIPTION
**Pull request by janmazak**

### CODE REVIEW

File: src/auxDataHashBuilder.h [hunk 1]
```diff
@@ -58,48 +58,48 @@ void auxDataHashBuilder_init(
...
+        const uint8_t* votePubKeyBuffer, size_t votePubKeySize,
         uint32_t weight
 );
-void auxDataHashBuilder_governanceVotingRegistration_addStakingKey(
+void auxDataHashBuilder_cVoteRegistration_addStakingKey(
         aux_data_hash_builder_t* builder,
         const uint8_t* stakingPubKeyBuffer, size_t stakingPubKeySize
 );
-void auxDataHashBuilder_governanceVotingRegistration_addVotingRewardsAddress(
+void auxDataHashBuilder_cVoteRegistration_addpaymentAddress(
```
[the remark this answers is not in the document]
Comment (davidmisiak) [reply src/auxDataHashBuilder.h:83]: Also in `signTxCVoteRegistration_handlepaymentAddressAPDU`

File: src/signTxCVoteRegistration.h [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	uint8_t auxDataHash[AUX_DATA_HASH_LENGTH];
+
+	union {
+		struct {
+			cvote_delegation_type_t type;
+			bip44_path_t votePubKeyPath;
+			uint8_t votePubKey[CVOTE_PUBLIC_KEY_LENGTH];
+			uint32_t weight;
+		} delegation;
+		tx_output_destination_storage_t rewardDestination;
```
Comment (davidmisiak) [inline src/signTxCVoteRegistration.h:56]: It's `rewardDestination` here, but the corresponding security policy calls it `paymentDestination`.

**Review by davidmisiak — APPROVED** [review]
Apart from the naming nits, LGTM. I didn't run anything (let me know if needed).

**Comment by janmazak** [comment]
Moved to https://github.com/LedgerHQ/app-cardano/pull/34

**Pull request closed without merging**